app/controllers/entitlements_controller.rb

```diff
@@ -19,4 +19,20 @@ def entitlement_search
   def entitlements
     @entitlements ||= Entitlement.all
   end
+
+  # Returns whether the dependency_status XHR parameter matches "status". It's
+  # "safe" because it also returns true if no XHR has been made. (It returns
+  # true instead of false so that the _entitlements_table always shows content.)
+  def safe_dependency_status_check(status)
```

Comment:
@jgarber623-gov Here are the helpers you suggested. Still kind of awkward, so I documented the intent and rationale.